media-libs/libdmtx/libdmtx-0.7.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Library to read and write Data Matrix barcodes"
+DESCRIPTION="libdmtx is a software library that enables programs to read and \
+write Data Matrix (or DataMatrix) 2D barcodes. It runs on several platforms, \
+and can be accessed by multiple languages."
+HOMEPAGE="http://libmtx.sourceforge.net/"
+COPYRIGHT="2005-2011 Mike Laughton"
```

Comment:
According to the last lines in ChangeLog, COPYRIGHT should be
```
COPYRIGHT="2006-2011 Mike Laughton"
```
but I might be missing something, so please ignore this comment if you know the project started in 2005.